test/integration/ServerFetch.test.ts

```diff
@@ -36,17 +36,50 @@ describe('A Solid server', (): void => {
     });
   });
 
+  it('can get the HEADers of a container.', async(): Promise<void> => {
```

Comment:
Test description does not seem to match the test.

Reply:
I don't really see the problem with it, unless you mean because the actual headers don't get checked it should be `can do a succesful HEAD request on a container`?

Reply:
Yes; you're not checking the headers in the test.